zerver/migrations/0001_initial.py

```diff
@@ -754,91 +638,103 @@ class Migration(migrations.Migration):
                     "create_time",
                     models.DateTimeField(db_index=True, default=django.utils.timezone.now),
                 ),
-                ("messages", models.ManyToManyField(to="zerver.Message")),
+                ("messages", models.ManyToManyField(to="zerver.message")),
                 (
                     "owner",
                     models.ForeignKey(
                         on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL
                     ),
                 ),
                 ("is_realm_public", models.BooleanField(default=False)),
+                (
+                    "realm",
+                    models.ForeignKey(
+                        blank=True,
+                        null=True,
+                        on_delete=django.db.models.deletion.CASCADE,
+                        to="zerver.realm",
+                    ),
+                ),
             ],
         ),
-        migrations.AddField(
-            model_name="realm",
-            name="create_stream_by_admins_only",
-            field=models.BooleanField(default=False),
+        migrations.CreateModel(
+            name="Subscription",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True, primary_key=True, serialize=False, verbose_name="ID"
+                    ),
+                ),
+                ("active", models.BooleanField(default=True)),
+                ("in_home_view", models.BooleanField(default=True, null=True)),
+                ("color", models.CharField(default="#c2c2c2", max_length=10)),
+                ("desktop_notifications", models.BooleanField(default=True)),
+                ("audible_notifications", models.BooleanField(default=True)),
+                ("notifications", models.BooleanField(default=False)),
+                (
+                    "recipient",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE, to="zerver.recipient"
+                    ),
+                ),
+                (
+                    "user_profile",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL
+                    ),
+                ),
+                ("pin_to_top", models.BooleanField(default=False)),
+            ],
+            options={
+                "unique_together": {("user_profile", "recipient")},
+            },
         ),
-        migrations.AddField(
+        # These indexes seem to not squash properly.
+        migrations.AddIndex(
```

Comment:
They should, though, since https://github.com/django/django/commit/8e2460d599aec95f8cfe514d3cc8acdd4ca4b1fb is in Django 5.0?

Does this work if you put them right after their models?